app/sentry.ts

```diff
@@ -0,0 +1,55 @@
+import * as Sentry from '@sentry/browser';
+import { CaptureConsole, Dedupe, Ember } from '@sentry/integrations';
+import config from 'open-event-frontend/config/environment';
+
+Sentry.init({
+    integrations: [
+        new Ember(),
+        new Dedupe(),
+        new CaptureConsole({
+            levels: ['error']
+        })
+    ],
+    beforeSend(event: Sentry.Event) {
+        const exception = event.exception?.values?.[0];
+        const errorValue = exception?.value;
+        if (errorValue?.includes("Ember Data Request")) {
+            if (errorValue?.includes("404")) {
+                // Ignore 404 errors from Ember Data because 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5114297)